docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '2.1'
+services:
+  web:
+    image: quay.io/mozmar/kitsune_base
```

Comment:
You don't need to define `image` when using `build`. Also what's ` quay.io/mozmar/kitsune_base`?